File: CSE221/lab4/A.cpp
#include <bits/stdc++.h>
using namespace std;

int main (){
    int node; 
    int edge; 
    cin >> node;
    cin >> edge;
    int arr[node*node];
    adj_matrix(node, edge, arr);
    for (int i=0; i < sizeof(arr) / sizeof(arr[0]); i++){
        for (int j=0; j < sizeof(arr) / sizeof(arr[0]); j++){
            cout << arr[i][j] << " ";
        }
    }

}

int adj_matrix(int node, int edge, int arr[]){

}

File: CSE221/lab4/A.py
def adj_mat(node, edge):
    arr = [[0]*node for _ in range(node)]
    for i in range(edge):
        num = list(map(int, input().split()))
        arr[num[0]-1][num[1]-1] = num[2]
    return arr
 
usr = list(map(int, input().split()))
res = adj_mat(usr[0], usr[1])
print('\n'.join(' '.join(map(str, row)) for row in res))